src/server/config/schema.js

```diff
@@ -161,6 +161,13 @@ module.exports = () => Joi.object({
   status: Joi.object({
     allowAnonymous: Joi.boolean().default(false)
   }).default(),
+  map: Joi.object({
+    manifestServiceUrl: Joi.when('$dev', {
```

Comment:
So the new `map.manifesetServiceUrl` is used for both the tiles and the geo layers, does that mean we should be deprecating the `tilemap` config section?